Apps/HNClient/src/appsettings.Development.json

```diff
@@ -5,5 +5,11 @@
             "System": "Debug",
             "Microsoft": "Information"
         }
-    }
+    },
+    "OpenIdConnect": {
+        "Authority": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/ff09qn3f",
+        "RequireHttpsMetadata": "false",
```

Comment:
This should be omitted now or changed to true.

Reply:
+1. Https should be necesessary for KetCloak

Reply:
agreed. will remove these settings as default is true